### DISCUSSION TITLE
Too many dependencies

### DISCUSSION THREAD

**Discussion by zain-sohail** [OpenCOMPES/sed#288] [General]
We had a short discussion about this in a PR

We should see what dependencies we can go away from. 

For example, I see that even though we don't include sphinx in our package directly, because symmetrize includes it, we end up installing it too. I guess there are many more such stuff.

**Comment by zain-sohail** [comment]
Like you mentioned @rettigl, it seems pynxtools has a big dependency list. And many of the things done there are not really relevant to us. I looked at the licensing and in principle, it's possible we take the convertor part of their code and just use that.
There are of course other packages we can think about that we don't necessarily need.

**Reply by rettigl** [reply]
We are working on disecting pynxtools into a core part, and plugins, which will reduce the number of dependencies dramatically. I suggest wating until this is sorted out.

**Comment by zain-sohail** [comment]
I saw that opencv-python is a dependency for sed but I couldn't find it being imported anywhere @rettigl
